cli/config/configfile/file_test.go

```diff
@@ -237,6 +239,48 @@ func TestGetAllCredentialsCredsStore(t *testing.T) {
 	assert.Check(t, is.Equal(1, testCredsStore.(*mockNativeStore).GetAllCallCount))
 }
 
+func TestGetAllCredentialsCredStoreErrorHandling(t *testing.T) {
+	const (
+		workingHelperRegistryHostname = "working-helper.com"
+		brokenHelperRegistryHostname  = "broken-helper.com"
```

Comment:
Sorry 🤦 jumping between etoomanythings; while these domains probably would never be actually hit, we should avoid using "fake" domains that (potentially) are not "fake". It's better to use domains that are designated for this purpose (see [RFC 2606 (section 2 and 3)](https://www.rfc-editor.org/rfc/rfc2606.html#section-2)). I personally tend to use `example.com` / `example.org` for these (although the `.test` TLD _SHOULD_ be reserved as well, but I've encountered setups where people used it as internal domains).

These could be (e.g.) `working-helper.example.com`,  `broken-helper.example.com`

Reply:
This is a good point! I tend to play it a bit loose with unit tests, but better safe than sorry.